src/Caller/MASA.Utils.Caller.DaprClient/DaprCallerProvider.cs

```diff
@@ -32,6 +37,7 @@ public override Task<TResponse> SendGrpcAsync<TResponse>(string methodName, Canc
     public override Task SendGrpcAsync<TRequest>(string methodName, TRequest request, CancellationToken cancellationToken = default)
         => _daprClient.InvokeMethodGrpcAsync(AppId, methodName, request, cancellationToken);
 
-    public override Task<TResponse> SendGrpcAsync<TRequest, TResponse>(string methodName, TRequest request, CancellationToken cancellationToken = default)
+    public override Task<TResponse> SendGrpcAsync<TRequest, TResponse>(string methodName, TRequest request,
+        CancellationToken cancellationToken = default)
```

Comment:
Why CancellationToken a single line

Reply:
Generated by the compiler formatting code, already processed